### PR TITLE
Update shell and vscode runners

### PR DESCRIPTION
**Pull request by timlinux**
Fixes issues with nix shell and vscode setup.

## Summary by CodeRabbit

- **Documentation**
	- Updated setup instructions for Hugo Dev Environment and integrated development tools.
- **Chores**
	- Enhanced development setup by automatically installing essential VS Code extensions for Hugo projects.

### CODE REVIEW

File: vscode.sh [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
-
+code --extensions-dir=.vscode-extensions --install-extension akmittal.hugofy
+code --extensions-dir=.vscode-extensions --install-extension arthaud-proust.hugo-partials-defs
+code --extensions-dir=.vscode-extensions --install-extension eliostruyf.vscode-hugo-themer
+code --extensions-dir=.vscode-extensions --install-extension hitkodev.vscode-remark-hugo
+code --extensions-dir=.vscode-extensions --install-extension kaellarkin.hugo-shortcode-syntax
```
[the remark this answers is not in the document]
Comment (timlinux) [reply vscode.sh:6]: This last line is correct, the bot is confused...

[the remark this answers is not in the document]
Comment (timlinux) [reply vscode.sh:6]: The last line will launch vscode using the locally installed extensions rather than the globally installed extensions

**Comment by timlinux** [comment]
@Xpirix can you take a look at this and indicate if I should merge regardless of the failure (given that I never changed anything that should impact the tests, and that you have an update for the e2e incoming)?

**Comment by Xpirix** [comment]
Hi @timlinux , I've tested this PR with the updated e2e tests and they passed: please see https://github.com/Xpirix/QGIS-Hugo/pull/10 for details. I think this not fail anymore once PR #205 is merged.

**Pull request merged**